content/sensu-go/5.17/api/overview.md

```diff
@@ -294,50 +292,167 @@ Here's the list of available fields:
+
+| operator | description     | example                |
+| -------- | --------------- | ---------------------- |
+| `==`     | Equality        | `check.publish == true`
+| `!=`     | Inequality      | `check.namespace != "default"`
+| `in`     | Included in     | `linux in check.subscriptions`
+| `notin`  | Not included in | `slack notin check.handlers`
+| `&&`     | Logical AND     | `check.publish == true && slack in check.handlers`
+
+#### Equality-based operators
+
+Sensu's two _equality-based_ operators are `==` (equality) and `!=` (inequality).
+
+For example, to retrieve only checks with the label `type` and value `server`: 
 
 {{< highlight shell >}}
-check.publish == true
-check.namespace != "default"
+curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
+--data-urlencode 'labelSelector=type == "server"'
 {{< /highlight >}}
 
-In addition, there are two _set-based_ operators to deal with lists of values: `in` and `notin`.
+_**NOTE**: Use the flag `--data-urlencode` in cURL to encode the query parameter. 
+Include the `-G` flag so the request appends the query parameter data to the URL._
+
+To retrieve checks that are not in the `production` namespace:
 
 {{< highlight shell >}}
-linux in check.subscriptions
-slack notin check.handlers
-check.namespace in [dev,production]
+curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
+--data-urlencode 'fieldSelector=check.namespace != "production"'
 {{< /highlight >}}
 
-### Combine selectors and statements
+#### Set-based operators
 
-You can combine multiple statements separated with the logical operator `&&` (_AND_) in field and label selectors.
+Sensu's two _set-based_ operators for lists of values are `in` and `notin`.
 
-For example, the following cURL request looks up checks that are configured to be published **and** include the `slack` handler:
+For example, to retrieve checks with a `linux` subscription:
 
 {{< highlight shell >}}
 curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
---data-urlencode 'fieldSelector=check.publish == true && slack in check.handlers'
+--data-urlencode 'linux in check.subscriptions'
 {{< /highlight >}}
 
-In addition to selectors with multiple statements, you can use field and label selectors at the same time:
+_**NOTE**: The `in` and `notin` operators only work when the underlying value is a string. You cannot use `in` or `notin` with integer, float, array, or Boolean values._
+
+To retrieve checks that do not use the `slack` handler:
 
 {{< highlight shell >}}
 curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
---data-urlencode 'fieldSelector=slack in check.handlers' \
---data-urlencode 'labelSelector=region != "us-west-1"'
+--data-urlencode 'slack notin check.handlers'
 {{< /highlight >}}
 
```

Comment:
```suggestion

```suggestion
The `in` and `notin` operators have two important conditions.
First, `in` and `notin` only work when the underlying value you're filtering for is a string.
You can filter for strings and arrays of strings with `in` and `notin`, but you cannot filter for integer, float, array, or Boolean values.

Second, if you're filtering for a string, the string must be to the **left** of the operator: `string [in|notin] selector`.
If you're filtering for an array of strings, the array must be to the **right** of the operator: `selector [in|notin] [string1,string2]`
```
```